### PR TITLE
Further Ubuntu cleanup + fixes for phone layout

### PR DESCRIPTION
**Pull request by Herrie82**

### CODE REVIEW

File: tests/keyboard-test/UbuntuKeyboard/Dummy.qml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-import QtQuick 2.0
-
-Item {
-    property bool available: true
-}
```
Comment (Tofee) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: We should only remove that if we remove the corresponding C++ code, I think. Same for qmldir just below.

File: qml/keys/ActionKey.qml [hunk 2]
```diff
@@ -52,9 +52,10 @@ CharKey {
         id: iconImage
         source: Qt.resolvedUrl("../images/" + formFactor + "/" + __icon + ".png")
         anchors.centerIn: parent
+		anchors.verticalCenterOffset: Units.gu(-0.25)
         visible: (label == "")
-       // width: units.gu(2.5)
-       // height: units.gu(2.5)
+        // width: units.gu(2.5)
+        height: parent.height * 0.7
```
Comment (Tofee) [inline qml/keys/ActionKey.qml]: You sure about that ? it seems a bit weird to have all the action keys being smaller than the rest of the keys of the row

File: tests/keyboard-test/UbuntuKeyboard/Dummy.qml (hunk 1 above)
Comment (Herrie82) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: I couldn't find any related reference to this "available" in the c++ code.

File: qml/keys/ActionKey.qml (hunk 2 above)
Comment (Herrie82) [inline qml/keys/ActionKey.qml]: It's the icon image on the button. Which I made 0.7 * the button height so it fits inside the button :)

File: tests/keyboard-test/UbuntuKeyboard/Dummy.qml (hunk 1 above)
Comment (Tofee) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: The corresponding C++ plugin is implemented by src/plugin/plugin.cpp ; my idea was that this code, today empty, could maybe prove itself useful at some point in time.

Comment (Herrie82) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: Well the C++ code doesn't check for the available currently anywhere. So we can keep the C++ files and remove the Ubuntu ones. Alternatively we can rename them to something else instead if that makes you happier ;)

Comment (Herrie82) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: So we call it LuneOSKeyboard instead of UbuntuKeyboard ;)

Comment (Tofee) [inline tests/keyboard-test/UbuntuKeyboard/Dummy.qml]: I'll be happy to rename them :)

File: qml/keys/ActionKey.qml (hunk 2 above)
Comment (Herrie82) [inline qml/keys/ActionKey.qml]: This makes sure that we don't really need the icons to be properly sized, so we can use the high resolution ones for example.

**Pull request merged**